fenix/app/src/main/java/org/mozilla/fenix/home/intent/DefaultBrowserIntentProcessor.kt

```diff
@@ -30,12 +29,6 @@ class DefaultBrowserIntentProcessor(
 
```

Comment:
Now that this is only for the re-engagement notification I think we should :
- update the class doc
- Rename this file/class
?